EIPS/eip-3675.md

```diff
@@ -0,0 +1,247 @@
+
+
+### PoW block processing
+
+PoW blocks that are descendants of any terminal PoW block **MUST NOT** be imported. This implies that a terminal PoW block will be the last PoW block in the canonical chain.
+
+
+### Block structure
+
+Beginning with `TRANSITION_BLOCK`, a number of previously dynamic block fields are deprecated by enforcing these values to instead be constants. Each block field listed in the table below **MUST** be replaced with the corresponding constant value.
+
+| Field | Constant value | Comment |
+|-|-|-|
+| **`ommersHash`** | `0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347` | `= Keccak256(RLP([]))` |
+| **`difficulty`** | `0` |  |
+| **`extraData`** | `b''` | `RLP(b'') = 0x80` |
+| **`mixHash`** | `0x0000000000000000000000000000000000000000000000000000000000000000` |  |
+| **`nonce`** | `0x0000000000000000` |  |
+| **`ommers`** | `[]` | `RLP([]) = 0xc0`  |
+
+*Note*: Logic and validity conditions of block fields that are *not* specified here **MUST** remain unchanged. Additionally, the overall block format **MUST** remain unchanged.
+
+
+### Block validity
+
+Beginning with `TRANSITION_BLOCK`, the block validity conditions **MUST** be altered by the following:
+* Remove verification of the block's **`difficulty`** value with respect to the difficulty formula.
+* Remove verification of the block's **`nonce`** and **`mixHash`** values with respect to the Ethash function.
+* Remove all validation rules that are evaluated over the list of ommers and each member of this list, except for enforcing this list to be empty.
+* Add verification of the fields noted in [block structure](#block-structure) as their specified constant value, including the enforcement of empty ommers list.
+* Add verification of block validity with respect to the PoS consensus rules, i.e. assert that there **MUST** be a corresponding event for the block to consider it valid. Such events are limited to the following:
+    * A `POS_CONSENSUS_VALIDATED` event considering the block or any of its descendants as valid.
+    * A `POS_CHAINHEAD_SET` event, including the `POS_BLOCK_FINALIZED` form of this event, designating the block or any of its descendants as the head of the canonical chain.
+
+*Note*: If one of the new rules fails then the block **MUST** be invalidated.
+
+*Note*: Validity rules that are not specified in the list above **MUST** remain unchanged.
+
+
+### Block and ommer rewards
+
+Beginning with `TRANSITION_BLOCK`, block and ommer rewards are deprecated. Specifically, the following actions **MUST** be taken:
+* Remove increasing the balance of the block's **`beneficiary`** account by the basic block reward.
```

Comment:
I think this is referred to just as `block reward` in most places.  Calling it `basic block reward` may lead to confusion since that phrase isn't commonly used.